src/clr-angular/layout/nav/providers/responsive-navigation.service.ts

```diff
@@ -10,7 +10,7 @@ import { Subject } from 'rxjs';
 import { ResponsiveNavCodes } from '../responsive-nav-codes';
 import { ResponsiveNavControlMessage } from '../responsive-nav-control-message';
 
-@Injectable()
+@Injectable({ providedIn: 'root' })
```

Comment:
in this context, `root` is the application consuming Clarity, right?

Reply:
Exactly. It declares the service in the root injector for the application.